godot-core/src/builtin/array.rs

```diff
@@ -492,17 +505,36 @@ impl<T: GodotType + FromGodot> Array<T> {
 }
 
 impl<T: GodotType + ToGodot> Array<T> {
-    /// Finds the index of an existing value in a sorted array using binary search. Equivalent of
-    /// `bsearch` in GDScript.
+    /// Finds the index of an existing value in a sorted array using binary search.
+    /// Equivalent of `bsearch` in GDScript.
     ///
-    /// If the value is not present in the array, returns the insertion index that would maintain
-    /// sorting order.
+    /// If the value is not present in the array, returns the insertion index that
+    /// would maintain sorting order.
     ///
-    /// Calling `binary_search` on an unsorted array results in unspecified behavior.
-    pub fn binary_search(&self, value: &T) -> usize {
+    /// Calling `bsearch` on an unsorted array results in unspecified behavior.
```

Comment:
Generally, we try to keep 120-145 characters, see [here](https://godot-rust.github.io/book/contribute/conventions.html#formatting).

It's not important enough to block this PR, but maybe think of it next time 🙂